### PR TITLE
Add support for runtime property in HostConfig 

### PR DESCRIPTION
**Pull request by vbhavsar**
This PR:
- Changes the exitCode type from Integer to Long
- Adds runtime property to HostConfig to support setting runtimes such as `nvidia`

### CODE REVIEW

**Comment by mattnworb** [comment]
is the stuff about exit codes related to #1053 ?

also, can you update the CHANGELOG file?

**Comment by mattnworb** [comment]
the change itself LGTM 👍

**Comment by caipre** [comment]
👍

**Comment by vbhavsar** [comment]
> is the stuff about exit codes related to #1053 ?
Yes. The mismatch in exit code was breaking code in helios.

> also, can you update the CHANGELOG file?
👍Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 9.0.0
```
Comment (johnflavin) [inline CHANGELOG.md]: I don't know that a bump all the way up to version `9.0.0` is required. `8.13.0` seems more appropriate to me.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+Released August 29 2018
```
Comment (johnflavin) [inline CHANGELOG.md]: Can you change this to "Not yet released"? The date should be set when the release is made.

**Comment by vbhavsar** [comment]
> ... I request one additional change from Integer to Long on ContainerStatus.exitCode.

@johnflavin That's already part of this PR:
https://github.com/spotify/docker-client/pull/1060/files#diff-47e61bfd4487ca3e3deda5a11d30c091R47

**Pull request merged**